src/lib/stitching/gravity/__tests__/stitching.test.ts

```diff
@@ -1102,6 +1107,43 @@ describe("gravity/stitching", () => {
           })
         )
       })
+
+      it("resolves the thumbnailImage field with a fallback image from representative artwork ID if imageURL is empty", async () => {
+        const { resolvers } = await getGravityStitchedSchema()
+        const { thumbnailImage } = resolvers.MarketingCollection
+
+        const parent = {
+          image_url: null,
+          representativeArtworkId: "representative-artwork-id",
+        }
+        const args = {}
+        const info = { mergeInfo: { delegateToSchema: jest.fn() } }
+
+        const imageData = {
+          image_url: "cat.jpg",
+        }
+
+        const artworkLoader = () =>
+          Promise.resolve({
+            images: [imageData],
+          })
+        const sharedContext = {
+          artworkLoader,
+        }
+
+        await thumbnailImage.resolve(parent, args, sharedContext, info)
+
+        expect(sharedContext).toHaveProperty("artworkLoader")
```

Comment:
**question**: I think this check will always pass, since you are explicitly adding artworkLoader to sharedContext [here](https://github.com/artsy/metaphysics/pull/4942/files#diff-9bdc788fbe3d15873b3600965e16a61d783e2e9257078688b5d7e6a1c58943aeR1131), am I right?

Reply:
Yes, I can remove it